HiPMobile/HiPMobile.Droid/Activities/MainActivity.cs

```diff
@@ -191,6 +199,8 @@ private void SetUpPermissions ()
 
         protected override void OnDestroy ()
         {
+            extendedLocationListener.Unregister();
```

Comment:
Move this to OnStop/OnPause as we need to stop listening to location updates as long as the activity is not active